### ISSUE TITLE
Expected a color. Got: currentColor

### ISSUE THREAD

**Issue by fuddl** [Compass/compass#1325]
'currentColor' should be an accepted color property:
see https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentColor_keyword

**Comment by cimmanon** [comment]
What do you propose Sass _do_ with the `currentColor` keyword?

``` css
body {
    color: black;
}

div {
    color: red;
}

.foo {
    color: currentColor;
}
```

What color is `.foo`?

**Comment by micahgodbolt** [comment]
currentColor is currently a CSS specification, and has pretty solid support. A better example use case would be

``` CSS
.foo {
color: grey;
border: 1px solid currentColor;
}
```

This would provide grey text with a grey border.

As far as **inheritance** goes, the currentColor isn't so much a variable as it is an indication to use the 'current color' of the element's text. So if .baz .foo {color: red}, you will end up having a red border for any .foo inside of .baz.

Another distinction is that currentColor is **not a defined value** that can be manipulated with Sass. You can't say darken(currentColor, 10%) because we don't know what the currentColor is until the webpage is rendered.

That being said there ARE a few places where compass support would be useful. 

``` CSS
/*Valid */
.mydiv {
  color: blue;
  background-image: linear-gradient(#ff0000, currentColor);
}
```

This gradient is valid, and it will be beautiful (not). But:

``` SCSS
// Throws Errors
.mydiv {
  color: blue;
  @include background-image(linear-gradient(#ff0000, currentColor));
}
```

This is not valid because of "Not a valid color stop: Sass::Script::String: currentColor". Compass tries to convert that into a color instead of passing it through directly. 

### TL:DR

We need to look at the places where Compass accepts color values as arguments, and make sure it will pass currentColor through without trying to first convert it to hex.

**Comment by chriseppstein** [comment]
I'm pretty sure this is fixed on master.

**Comment by micahgodbolt** [comment]
@chriseppstein you could have said that before I wrote my tirade :)

**Comment by chriseppstein** [comment]
Sorry! FWIW, we agree :thumbsup:

**Comment by micahgodbolt** [comment]
@chriseppstein woot! I done good!

**Comment by scottdavis** [comment]
can someone verify that this is fixed on master so we can close this issue?
